bit9PlatformAPI/examples/connectors/Metascan.py

```diff
@@ -0,0 +1,247 @@
+"""
+First off, I'm not a programmer so any issues that you have in running this script in your environment are not my fault!
+I've borrowed HEAVILY from the script written for VirusTotal so I'm going to leave IN the disclaimer that Bit9 placed on
+the original script. Hopefully this will indemnify both Bit9 and me from any issues encountered by you if you use this
+script.
+
+If you can find a better way to work any of this, go for it. I'd ask that you update the script in GitHub if you do though.
+Thanks and enjoy!
+
+-Patrick
+
+Disclaimer
++++++++++++++++++++
+By accessing and/or using the samples scripts provided on this site (the "Scripts"), you hereby agree to the following terms:
+The Scripts are exemplars provided for purposes of illustration only and are not intended to represent specific
+recommendations or solutions for API integration activities as use cases can vary widely.
+THE SCRIPTS ARE PROVIDED "AS IS" WITHOUT WARRANTIES OF ANY KIND, EITHER EXPRESS OR IMPLIED.  BIT9 MAKES NO REPRESENTATION
+OR OTHER AFFIRMATION OF FACT, INCLUDING BUT NOT LIMITED TO STATEMENTS REGARDING THE SCRIPTS' SUITABILITY FOR USE OR PERFORMANCE.
+IN NO EVENT SHALL BIT9 BE LIABLE FOR SPECIAL, INCIDENTAL, CONSEQUENTIAL, EXEMPLARY OR OTHER INDIRECT DAMAGES OR FOR DIRECT
+DAMAGES ARISING OUT OF OR RESULTING FROM YOUR ACCESS OR USE OF THE SCRIPTS, EVEN IF BIT9 IS ADVISED OF OR AWARE OF THE
+POSSIBILITY OF SUCH DAMAGES.
+
+Requirements
++++++++++++++++++++
+
+- Bit9 API client (included) which requires requests Python module
+
+- Metascan API Key
+
+- Bit9 Platform Server 7.2.1 or later
+- Bit9 API Token (generated in Bit9 Console)
+- Bit9 RBAC permission enabled for 'Extend connectors through API'
+
+Required python modules can be installed using tools such as easy_install or pip, e.g.
+    easy_install requests
+
+Configuring Connector
++++++++++++++++++++++++
+
+Please update the script with appropriate vt_token, Bit9 server address and Bit9 token at the bottom of the script.
```

Comment:
Change this to reflect you need to change the `ms_token` and provide the hostname for your metascan server (see comments below)